app/apps/server/bridges/http.ts

```diff
@@ -78,16 +78,33 @@ export class AppHttpBridge extends HttpBridge {
 				url: info.url,
 				method: info.method,
 				statusCode: response.status,
-				data: await response.json(),
 				headers: Object.fromEntries(response.headers as unknown as any),
 			};
 
-			if (request.hasOwnProperty('encoding')) {
-				if (request.encoding === null) {
-					result.content = Buffer.from(await response.arrayBuffer()).toString();
-				} else {
-					result.content = await response.text();
-				}
+			const body = Buffer.from(await response.arrayBuffer());
+
+			if (request.encoding === null) {
```

Comment:
maybe you can check if the encoding is not null by: 

`if (!request.encoding)`

Reply:
Here we need a strict comparison, as `undefined` for example, is a valid parameter to the `Buffer.toString` function